packages/@expo/fingerprint/cli/src/commands/diffFingerprints.ts

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+import chalk from 'chalk';
+
+import { diffFingerprints } from '../../../build/index.js';
+import { Command } from '../cli.js';
+import { assertArgs, getFileArgumentAtIndex } from '../utils/args.js';
+import { CommandError } from '../utils/errors.js';
+import * as Log from '../utils/log.js';
+import readFingerprintFileAsync from '../utils/readFingerprintFileAsync.js';
```

Comment:
```suggestion
import { diffFingerprints } from '../../../build/index';
import { Command } from '../cli';
import { assertArgs, getFileArgumentAtIndex } from '../utils/args';
import { CommandError } from '../utils/errors';
import * as Log from '../utils/log';
import readFingerprintFileAsync from '../utils/readFingerprintFileAsync';
```

maybe not having the `.js` suffix because we are not fully migrated to ESM yet.